ExerciseOOP/exercise1.rb

```diff
@@ -0,0 +1,35 @@
+class Person
+  attr_accessor :name, :age
+
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def display
+    puts "Detail:"
```

Comment:
string không nội suy thì nên dùng single-quote nhé
```suggestion
    puts 'Detail:'
```